x-pack/plugin/sql/licenses/antlr4-runtime-4.5.3.jar.sha1

```diff
@@ -1 +0,0 @@
-2609e36f18f7e8d593cc1cddfb2ac776dc96b8e0
```

Comment:
Why remove these?

Reply:
Because of the `extended-plugin` functionality, the painless classpath is available to SQL and that already contains antlr. Which is great since it's one less jar (and its version) we need to worry about.

Reply:
Hmmmm. I don't particularly like that we inherit Painless's antlr but I suppose I have to live with it.